lib/rubocop/cop/style/string_concatenation.rb

```diff
@@ -11,16 +11,24 @@ module Style
       # In those cases, it might be useful to extract statements to local
       # variables or methods which you can then interpolate in a string.
       #
-      # @example
+      # @example EnforcedStyle: all (default)
```

Comment:
I'd expand the cop docs to explain the safety issues and the implications of the using the different styles.